src/components/posts/PostItem.js

```diff
@@ -67,14 +70,25 @@ const PostItem = ({ post, userId, setPostResponse, setPost, setProfileView }) =>
     navigate('/profile')
   }
 
+  const handleLike = (e) => {
+    setIsLiked(e.target.checked)
+
+    if(!isLiked) {
+      console.log('im liking it')
+    }
+    else {
+      console.log('im not liking it')
+    }
```

Comment:
It looks like this is either residual debugging code or placeholder for future functionality - let's remove these logs before merging this so it doesn't go into the live app